src/toil/common.py

```diff
@@ -639,79 +813,90 @@ def addOptions(parser: ArgumentParser, config: Optional[Config] = None, jobstore
                                           f"provided as a comma-separated list.  The first value is used as a default "
                                           f"if the list length is less than the number of nodeTypes.  "
-                                          f"default={config.maxNodes[0]}")
-    autoscaling_options.add_argument("--targetTime", dest="targetTime", default=None,
+                                          f"default={10}")
+    autoscaling_options.add_argument('--maxNodes', default=[10], dest="maxNodes", type=parse_int_list,
+                                     help=argparse.SUPPRESS)
+    autoscaling_options.add_argument("--target_time", "--targetTime", dest="targetTime", default=defaultTargetTime, type=int, action=make_closed_interval_check_action(0),
                                      help=f"Sets how rapidly you aim to complete jobs in seconds. Shorter times mean "
                                           f"more aggressive parallelization. The autoscaler attempts to scale up/down "
                                           f"so that it expects all queued jobs will complete within targetTime "
-                                          f"seconds.  default={config.targetTime}")
-    autoscaling_options.add_argument("--betaInertia", dest="betaInertia", default=None,
+                                          f"seconds.  default={defaultTargetTime}")
+    autoscaling_options.add_argument("--beta_inertia", "--betaInertia", dest="betaInertia", default=0.1, type=float, action=make_closed_interval_check_action(0.0, 0.9),
                                      help=f"A smoothing parameter to prevent unnecessary oscillations in the number "
                                           f"of provisioned nodes. This controls an exponentially weighted moving "
                                           f"average of the estimated number of nodes. A value of 0.0 disables any "
                                           f"smoothing, and a value of 0.9 will smooth so much that few changes will "
-                                          f"ever be made.  Must be between 0.0 and 0.9.  default={config.betaInertia}")
-    autoscaling_options.add_argument("--scaleInterval", dest="scaleInterval", default=None,
+                                          f"ever be made.  Must be between 0.0 and 0.9.  default={0.1}")
+    autoscaling_options.add_argument("--scale_interval", "--scaleInterval", dest="scaleInterval", default=60, type=int,
                                      help=f"The interval (seconds) between assessing if the scale of "
-                                          f"the cluster needs to change. default={config.scaleInterval}")
-    autoscaling_options.add_argument("--preemptibleCompensation", "--preemptableCompensation", dest="preemptibleCompensation", default=None,
+                                          f"the cluster needs to change. default={60}")
```

Comment:
I think we want to use `%(default)s` in these messages instead of putting the default value again.